docs/requirements/requirements.md

```diff
@@ -0,0 +1,112 @@
+# Hardware requirements of TwPM module
+
+[tcg_client_v13r27]:<https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClientTPMInterfaceSpecification_TIS__1-3_27_03212013.pdf>
+[tcg_client_PTP_v1p5r14]:<https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p05p_r14_pub.pdf>
+
+## Acronyms
+
+Legend:
+
+* PTP – Platform TPM Profile
+* CRB – Command Response Buffer interface
+* DDWG – Device Driver’s Writers Guide
+* Certification PP – Certification Protection Profile
+* TIS – TPM Interface Specification
```

Comment:
Some acronims are listed here, but not used anywhere in the document.

Reply:
Removed in https://github.com/Dasharo/twpm-docs/pull/4/commits/13c295be0672db6c329a6d93dc68ca54cd8ed746